sdk/identity/identity/Troubleshooting.md

```diff
@@ -164,7 +164,7 @@ The Managed Identity credential runs only on Azure-hosted machines/servers. Ensu
 | [Azure Virtual Machines](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token) | [Configuration Instructions](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/qs-configure-portal-windows-vm)          |
 | [Azure App Service](https://docs.microsoft.com/azure/app-service/overview-managed-identity?tabs=javascript)                        | [Configuration Instructions](https://docs.microsoft.com/azure/app-service/overview-managed-identity?tabs=java)                                             |
 | [Azure Kubernetes Service](https://docs.microsoft.com/azure/aks/use-managed-identity)                                              | [Configuration Instructions](https://docs.microsoft.com/azure/aks/use-managed-identity)                                                                    |
-| [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/msi-authorization)                                                |                                                                                                                                                            |
+| [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/msi-authorization)                                                | [Configuration Instructions](https://docs.microsoft.com/en-us/azure/cloud-shell/msi-authorization)                                                         |
```

Comment:
Should the column say "N/A" instead? According to the last bullet [here](https://docs.microsoft.com/azure/cloud-shell/msi-authorization#limitations), managed identities aren't available in Cloud Shell.

Reply:
Thanks! I updated it to "Not Applicable"